app/js/pages/Profile/index.jsx

```diff
@@ -29,7 +31,16 @@ class Profile extends Component {
         <h3 className="mt-3">{profile.location}</h3>
         <Map coords={{latitude: 45.492611, longitude: -73.617959}} markerOnly={true}/>
         <p className="text-muted mt-2">Saalestraße 39A, 12055 Berlin</p>
-        <Offers offers={profile.offers} onClick={this.offerClick}/>
+        <Row>
+          <Col xs="12" className="mt-2">
```

Comment:
IMO, doing a row just for one col doesn't do much, it's the same as a normal div at this point. Am I missing something?

Reply:
Yes, Row are removing the padding from the container, so the background is full width

Reply:
Ah ok, understood